src/index.ts

```diff
@@ -34,25 +34,24 @@ export async function resolve(
   context: ResolveContext,
   defaultResolve: Function,
 ): Promise<ResolveResponse> {
-  let { parentURL = baseURL } = context;
+  const { parentURL = baseURL } = context;
 
-  let forceRelative = false;
+  // let forceRelative = false;
   if (TSConfig?.paths) {
     for (const tsPath of Object.keys(TSConfig.paths)) {
-      const tsPathKey = tsPath.replace('/*', '');
+      const tsPathKey = tsPath.replace('*', '');
```

Comment:
I feel like there could be a way to replace the `*` in tsPath and the path itself along with replacing the specifier, I tried a few ways to do it, but I can't find the flow/logic.

What I want it to do is somehow replace the specifier, tsPath and result in a single regex or statement.